app/Http/Controllers/ProjectController.php

```diff
@@ -106,7 +106,7 @@ public function update(Request $request, Project $project)
             'short_description' => $request->short_description,
             'long_description'  => $request->long_description,
             'github_link'       => $request->github_link,
-            'website_link'      => $request->website_link,
+            'siteweb_link'      => $request->siteweb_link,
```

Comment:
Website is english, siteweb is french. So I think it's better to keep website and change in all files siteweb_link to website_link